install_deps.sh

```diff
@@ -10,7 +11,7 @@ BASE_FLAGS="-fPIC -O3 -g"
 
 PLATFORM=$(uname -s)
 if [ "$PLATFORM" == "Darwin" ]; then
-    C_FLAGS="$BASE_FLAGS -mmacosx-version-min=10.7 -arch i386"
+    C_FLAGS="$BASE_FLAGS -mmacosx-version-min=10.7 -arch x86_64"
```

Comment:
this is the only line in this file that is strictly necessary to fix for MacOSX -- all the rest of the diff in `install_deps.sh` here i just found more useful, but it could all be dropped.